Sage/carbon
# fix(storybook): fix theme selector functionality - FE-5004

### Proposed behaviour
This PR fixes an issue with ThemeSelector which does not change the theme immediately and a manual page refresh is required.

### Current behaviour
![Zrzut ekranu 2022-03-3 o 17 25 24](https://user-images.githubusercontent.com/20651022/156607309-e9c48260-8244-4c28-9cc8-3c962b2ae2f8.png)

### Checklist




#### QA


### Testing instructions

Run storybook locally and see if changing theme in the toolbar results in an immediate theme change
